test/unit/puppetlabs/services/ca/certificate_authority_core_test.clj

```diff
@@ -305,6 +305,20 @@
           (is (= #{localhost-status test-agent-status revoked-agent-status}
                  (set (json/parse-string (:body response) true)))))
 
+        (testing "requires ignored path segment"
+          (let [response (test-app
+                          {:uri "/v1/certificate_statuses/"
+                           :request-method :get})]
+            (is (= 404 (:status response)))))
```

Comment:
This seems more like a 400 than a 404.  But that [doesn't seem to be documented](https://docs.puppetlabs.com/puppet/latest/reference/http_api/http_certificate_status.html) and I'm not sure if it would be worth the effort to implement that change.  Thoughts?  I'm going to merge this as-is but feel free to submit a follow-up PR if you think it's worth the effort to switch to a 400 here.